src/index.ts

```
@@ -605,6 +605,16 @@ export namespace OneSignal {
 
       return RNOneSignal.getTags();
     }
+
+    /** Track custom events for the current user. */
+    export function trackEvent(
+      name: string,
+      properties?: Record<string, string>,
+    ) {
+      if (!isNativeModuleLoaded(RNOneSignal)) return;
```

Should this silently return, or at least warn the user that `RNOneSignal` is not loaded?

---

The pattern is the same for all the methods. The isNativeModuleLoaded method logs an error.